react/features/base/tracks/middleware.js

```diff
@@ -108,30 +106,23 @@ MiddlewareRegistry.register(store => next => action => {
             const participantID = jitsiTrack.getParticipantId();
             const isVideoTrack = jitsiTrack.isVideoTrack();
 
-            if (jitsiTrack.isLocal()) {
-                if (isVideoTrack) {
+            if (isVideoTrack) {
+                if (jitsiTrack.isLocal()) {
                     APP.conference.setVideoMuteStatus(muted);
                 } else {
-                    APP.conference.setAudioMuteStatus(muted);
+                    APP.UI.setVideoMuted(participantID, muted);
                 }
-            }
-
-            if (isVideoTrack) {
-                APP.UI.setVideoMuted(participantID, muted);
                 APP.UI.onPeerVideoTypeChanged(
                     participantID,
                     jitsiTrack.videoType);
             } else {
-                APP.UI.setAudioMuted(participantID, muted);
+                // eslint-disable-next-line no-lonely-if
```

Comment:
I find it easier to read the other way. Generally, I try not to bend the eslint rules unless they is a strong case for it. I won't fight you to death about this though!